CONTRIBUTING.md

```diff
@@ -0,0 +1,136 @@
+
+```
+$ git clone github.com/<USERNAME_GH>/<SLUG_GH>.git
+$ cd <SLUG_GH>
+$ git remote add fork git://github.com/yourusername/<SLUG_GH>.git
+```
+
+#### Step 2: Branch
+
+Create a branch and start hacking:
+
+```
+$ git checkout -b my-branch origin/master
+```
+
+#### Step 3: Code
+
+Well, we think you know how to do that. Just be sure to follow the coding guidelines from the community ([standard JS][stdjs], comment the code, etc).
+
+#### Step 4: Test
+
+Don't forget to add tests and be sure they are green:
+
+```
+$ cd <SLUG_GH>
+$ npm run test
+```
+
+#### Step 5: Commit
+
+Writing [good commit messages][commitmsg] is important. A commit message should describe what changed and why.
+
+#### Step 6: Rebase
+
+Use `git rebase` (_not_ `git merge`) to sync your work from time to time.
+
+```
+$ git fetch origin
+$ git rebase origin/master my-branch
+```
+
+#### Step 7: Push
+
+```
+$ git push -u fork my-branch
+```
+
+Go to https://github.com/yourusername/<SLUG_GH> and select your branch. Click the 'Pull Request' button and fill out the form.
+
+Alternatively, you can use [hub] to open the pull request from your terminal:
+
+```
+$ git pull-request -b master -m "My PR message" -o
+```
+
+Pull requests are usually reviewed within a few days. If there are comments to address, apply your changes in a separate commit and push that to your branch. Post a comment in the pull request afterwards; GitHub doesn't send out notifications when you add commits.
+
+
+Writing documentation
+---------------------
+
+Documentation improvements are very welcome. We try to keep a good documentation in the `/docs` folder. But, you know, we are developers, we can forget to document important stuff that look obvious to us. And documentation can always be improved.
+
+
+Translations
+------------
+
+The Cozy <APP_NAME> is translated on a platform called [Transifex][tx]. [This tutorial][tx-start] can help you to learn how to make your first steps here. If you have any question, don't hesitate to ask us!
+
+
+Community
+---------
+
+You can help us by making our community even more vibrant. For example, you can write a blog post, take some videos, answer the questions on [the forum][forum], organize new meetups, and speak about what you like in Cozy!
+
+
+
+[issues]: https://github.com/<USERNAME_GH>/<SLUG_GH>/issues/new
+[checkbox]: https://github.com/blog/1375-task-lists-in-gfm-issues-pulls-comments
+[pr]: https://help.github.com/categories/collaborating-with-issues-and-pull-requests/
+[forking]: http://blog.campoy.cat/2014/03/github-and-go-forking-pull-requests-and.html
+[stdjs]: http://standardjs.com/
+[commitmsg]: http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html
+[localization]: https://github.com/<USERNAME_GH>/<SLUG_GH>/blob/master/README.md#localization
+[hub]: https://hub.github.com/
+[tx]: https://www.transifex.com/cozy/
+[tx-start]: http://docs.transifex.com/getting-started/translators/
+[forum]: https://forum.cozy.io/
```

Comment:
No change here from `cozy-konnector-template`.

Reply:
You should change all  the `<FIELDS>`

Reply:
Oh, right, I'll do it.

Reply:
Done.